### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by b00f** [herumi/bls#89]
The `minimized_static` target in Makefile has error. I tried to build like: `MIN_WITH_XBYAK=1 make minimized_static;` but I got another error. 
Any idea how to fix it?

You can check the log here: https://github.com/zarbchain/zarb-go/runs/7745122863?check_suite_focus=true#step:5:233 

Actual command is like this: 
```
HERUMI= $(shell pwd)/.herumi
git clone --recursive https://github.com/herumi/bls.git $(HERUMI)/bls && cd $(HERUMI)/bls && make minimized_static;
```

**Comment by herumi** [comment]
Thank you for the report. I'll fix it.
By the way, what environments do you want to use `minimized_static`?

**Comment by herumi** [comment]
If you need a static library including libmcl.a, then

```
make -f Makefile.onelib
```
generates it.

**Comment by b00f** [comment]
>  By the way, what environments do you want to use minimized_static?

Inside docker (Alpine-Linux)

**Comment by herumi** [comment]
I've fixed it at v1.28, but as I wrote at https://github.com/herumi/bls/issues/89#issuecomment-1210179994 , make -f Makefile.onelib is better.

**Comment by b00f** [comment]
@herumi thanks for the fix.
